source/common/ssl/context_config_impl.h

```diff
@@ -39,16 +39,27 @@ class ContextConfigImpl : public virtual Ssl::ContextConfig {
 
   bool isReady() const override {
     // Either tls_certficate_provider_ is nullptr or
-    // tls_certficate_provider_->secret() is NOT nullptr.
-    return !tls_certficate_provider_ || tls_certficate_provider_->secret() != nullptr;
+    // tls_certficate_provider_->secret() is NOT nullptr and
+    // either certficate_validation_context_provider_ is nullptr or
+    // certficate_validation_context_provider_->secret() is NOT nullptr.
+    return (!tls_certficate_provider_ || tls_certficate_provider_->secret() != nullptr) &&
+           (!certficate_validation_context_provider_ ||
+            certficate_validation_context_provider_->secret() != nullptr);
   }
 
   void setSecretUpdateCallback(std::function<void()> callback) override {
     if (tls_certficate_provider_) {
-      if (secret_update_callback_handle_) {
-        secret_update_callback_handle_->remove();
+      if (tc_update_callback_handle_) {
+        tc_update_callback_handle_->remove();
```

Comment:
Missing coverage, https://95081-65214191-gh.circle-artifacts.com/0/build/envoy/generated/coverage/coverage.bazel-out_k8-dbg_bin_source_common_ssl__virtual_includes_context_config_lib_common_ssl_context_config_impl.h.html

Reply:
Fixed in context_impl_test.cc. Thanks.

Reply:
Fixed in context_impl_test.cc. Thanks.